tests/api/cluster/test_bucket.py

```diff
@@ -0,0 +1,23 @@
+from model_mommy import mommy
+import pytest
+
+from controlpanel.api import cluster
+
+
+@pytest.fixture(autouse=True)
+def enable_db_for_all_tests(db):
+    pass
+
+
+@pytest.fixture
+def bucket():
+    return mommy.make('api.S3Bucket', name='test-bucket')
```

Comment:
as mentioned in other comment, probably persist the `S3Bucket` to the DB just to test the `arn` property and check the AWS calls are called is overkill and unnecessarily slow down the tests.